docs/manual/format.md

```diff
@@ -85,88 +88,72 @@ package, padded with null (0) bytes.
 00000048: 00 00 00 00 00 01 00 05    ........
 ```
 
-Bytes 76-77 ("00 01" above) form an int16 that indicates the OS the
-package was built for.  In this case, 1 == Linux.  The next 2 bytes
-(78-79) form an int16 that indicates the signature type.  This tells
-RPM what to expect in the Signature.  For version 3.0 packages, this
-is 5, which indicates the new "Header-style" signatures.
+Bytes 76-77 ("00 01" above) form an int16 that indicates the OS the package was
+built for.  In this case, 1 == Linux.  The next 2 bytes (78-79) form an int16
+that indicates the signature type.  This tells RPM what to expect in the
+Signature.  This is generally expected to be 5, which indicates the use of
+"Header-style" signatures.
 
 ```
 00000050: 04 00 00 00 68 e6 ff bf    ........
 00000058: ab ad 00 08 3c eb ff bf    ........
 ```
 
-The remaining 16 bytes (80-95) are currently unused and are reserved
-for future expansion.
+The remaining 16 bytes (80-95) are unused.
 
 ## Signature
 
-A 3.0 format signature (denoted by signature type 5 in the Lead), uses
-the same structure as the Header.  For historical reasons, this
-structure is called a "header structure", which can be confusing since
-it is used for both the Header and the Signature.  The details of the
-header structure are given below, and you'll want to read them so the
-rest of this makes sense.  The tags for the Signature are defined in
-lib/signature.h.
-
-The Signature can contain multiple signatures, of different types.
-There are currently only three types, each with its own tag in the
-header structure:
-
-```
-	Name	Tag	Header Type
-	----	----	-----------
-	SIZE	1000	INT_32
-	MD5	1001	BIN
-	PGP	1002	BIN
-```
-
-The MD5 signature is 16 bytes, and the PGP signature varies with
-the size of the PGP key used to sign the package.
-
-As of RPM 2.1, all packages carry at least SIZE and MD5 signatures,
-and the Signature section is padded to a multiple of 8 bytes.
+"Header-style" signatures (denoted by signature type 5 in the Lead), use the
```

Comment:
Unsure if "denoted by" should stay - at this point, it should probably be assumed, regardless of what the lead says.  No reason to look at the lead at all.

Reply:
Rpm doesn't look at the lead except for the magic, so it's totally irrelevant. Even rpm v3 only supported header-style signatures IIRC.